src/user/serializers.py

```diff
@@ -148,6 +150,12 @@ def get_num_posts(self, author):
             return ResearchhubPost.objects.filter(created_by=user).count()
         return 0
 
+    def get_status(self, author):
+        return {
```

Comment:
if this is `suspended_status`, then we should return a True / False here. So then the logic is dictated by the backend instead of being split between the FE / BE

Reply:
The main issue is that I am currently differentiating suspension from banned. Maybe we don't need to?